index.bs

```diff
@@ -3453,6 +3453,14 @@ When this operation is invoked, the [=authenticator=] MUST perform the following
             ::  return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
         </dl>
 
+        Note: The purpose of this [=authorization gesture=] is not to proceed with creating a credential,
+        but for privacy reasons to authorize disclosure of the fact that
+        <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is [=bound credential|bound=] to this [=authenticator=].
+        If the user consents, the [=client=] and [=[RP]=] can detect this and guide the user to use a different [=authenticator=].
+        If the user does not consent,
+        the [=authenticator=] does not reveal that <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is bound to it,
+        and responds as if the user simply declined consent to create a credential.
+
 1. If |requireResidentKey| is [TRUE] and the authenticator cannot store a [=client-side-resident public key credential source=],
```

Comment:
diff --git a/index.bs b/index.bs
index a36b66cf..55a2162e 100644
--- a/index.bs
+++ b/index.bs
@@ -3453,6 +3453,15 @@ When this operation is invoked, the [=authenticator=] MUST perform the following
             ::  return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
         </dl>
 
+        Note: The purpose of this [=authorization gesture=] is not to proceed with creating a credential,
+        but for privacy reasons to authorize disclosure of the fact that
+        <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is [=bound credential|bound=] to this [=authenticator=].
+        If the user consents, the [=client=] and [=[RP]=] can detect this and guide the user to use a different [=authenticator=].
+        If the user does not consent,
+        the [=authenticator=] does not reveal that <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code>
+        is [=bound credential|bound=] to it,
+        and responds as if the user simply declined consent to create a credential.
+
 1. If |requireResidentKey| is [TRUE] and the authenticator cannot store a [=client-side-resident public key credential source=],
     return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
 1. If |requireUserVerification| is [TRUE] and the authenticator cannot perform [=user verification=], return an error code

Reply:
From 50a22e4bb3730c1e83f2503c52511eaedffb2a0e Mon Sep 17 00:00:00 2001
From: Emil Lundberg <emil@yubico.com>
Date: Fri, 18 Oct 2019 16:58:23 +0200
Subject: [PATCH 1/2] Add explanatory note to step 3 in
 authenticatorMakeCredential

---
 index.bs | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/index.bs b/index.bs
index a36b66cf..a6fea5a4 100644
--- a/index.bs
+++ b/index.bs
@@ -3453,6 +3453,14 @@ When this operation is invoked, the [=authenticator=] MUST perform the following
             ::  return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
         </dl>
 
+        Note: The purpose of this [=authorization gesture=] is not to proceed with creating a credential,
+        but for privacy reasons to authorize disclosure of the fact that
+        <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is [=bound credential|bound=] to this [=authenticator=].
+        If the user consents, the [=client=] and [=[RP]=] can detect this and guide the user to use a different [=authenticator=].
+        If the user does not consent,
+        the [=authenticator=] does not reveal that <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is bound to it,
+        and responds as if the user simply declined consent to create a credential.
+
 1. If |requireResidentKey| is [TRUE] and the authenticator cannot store a [=client-side-resident public key credential source=],
     return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
 1. If |requireUserVerification| is [TRUE] and the authenticator cannot perform [=user verification=], return an error code

From b58f3a308be5d26e7401ca7813da2ccfebd17d47 Mon Sep 17 00:00:00 2001
From: Emil Lundberg <emil@yubico.com>
Date: Mon, 21 Oct 2019 12:31:00 +0200
Subject: [PATCH 2/2] Add definition link to "bound" term

---
 index.bs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/index.bs b/index.bs
index a6fea5a4..55a2162e 100644
--- a/index.bs
+++ b/index.bs
@@ -3458,7 +3458,8 @@ When this operation is invoked, the [=authenticator=] MUST perform the following
         <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is [=bound credential|bound=] to this [=authenticator=].
         If the user consents, the [=client=] and [=[RP]=] can detect this and guide the user to use a different [=authenticator=].
         If the user does not consent,
-        the [=authenticator=] does not reveal that <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is bound to it,
+        the [=authenticator=] does not reveal that <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code>
+        is [=bound credential|bound=] to it,
         and responds as if the user simply declined consent to create a credential.
 
 1. If |requireResidentKey| is [TRUE] and the authenticator cannot store a [=client-side-resident public key credential source=],

Reply:
This looks like accidental noise to me. Let me know if it isn't.